raiden/utils/debugging.py

```diff
@@ -38,6 +45,16 @@ def on_signal(signalnum: Any, stack_frame: Any) -> None:  # pylint: disable=unus
     signal.signal(signal.SIGUSR1, on_signal)
 
 
+def debug_asyncio() -> None:
+    try:
+        asyncio.get_running_loop()
+    except RuntimeError:
+        return
+    tasks = [task for task in asyncio.all_tasks() if task is not asyncio.current_task()]
```

Comment:
Is it worth it to save `asyncio.current_task()` into a variable and avoid calling the method every time?

Reply:
We should probably also avoid creating the list only to iterate over it -- we can just iterate, e.g.:
```python
for task in asyncio.all_tasks():
    if task is not asyncio.current_task():
        task.print_stack()
```

Reply:
Why do we exclude the `current_task()`?